app/controllers/carts_controller.rb

```diff
@@ -1,9 +1,11 @@
-class CartsController < ApplicationController 
+class CartsController < ApplicationController
+  before_action :logged_in_user, only: :show
+
   def index
-    @carts = Cart.all 
+    @carts = Cart.all
```

Comment:
Does this mean that if a user uses this method he can see all carts? for all other users?

Reply:
It doesn't even have a route for this, so technically, no ;)). I will remove it